.cirrus.yml

```diff
@@ -14,6 +14,7 @@ env:
     GOPATH: &gopath "/var/tmp/go"
     GOCACHE: "${GOPATH}/cache"
     GOSRC: &gosrc "/var/tmp/go/src/github.com/containers/podman"
+    CIRRUS_DEFAULT_WORK: ${CIRRUS_WORKING_DIR}
```

Comment:
What it's doing here is backing up the old Cirrus value before it's overwritten right after with a Unix path. Do you know of a way to refer to env variables that are pre-shadowed in Cirrus (e.g. something like ${input.CIRRUS_WORKING_DIR}? I figured there must be but couldn't find much searching around. I could move it to the task if something like that exists.

Reply:
OIC.  No and there's a bit of YAML-parsing magic happening here too (the standard is: mapping keys are un-ordered, but Cirrus takes them in order).  It might be worth adding a comment here to make the purpose more clear.  Something simple would be fine, like `# Some tasks need to use the default workdir path`